apps/condo/lang/ru/ru.json

```diff
@@ -772,6 +772,7 @@
   "pages.organizations.tin": "ИНН",
   "pages.organizations.tin.TooShortMessage": "ИНН должен содержать 10 символов",
   "pages.organizations.tin.InvalidValue": "Некорректное значение ИНН",
+  "pages.organizations.name.InvalidName": "Некорректное название",
```

Comment:
Not user-friendly message, how should I understand, what's wrong?

Reply:
It’s absolutely unfriendly. You should write a customer friendly messages

Reply:
fixed